Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -2003,6 +2008,10 @@ Cursor const& WindowManager::active_cursor() const
         return *m_move_cursor;
 
     if (m_resize_window || m_resize_candidate) {
+        if (m_resize_candidate && !m_resize_candidate->is_resizable()) {
+            return *m_disallowed_cursor;
+        }
```

Comment:
Hm, I'm not too fond of this change. Previously, it was already clear to the user which edges are resizable and which are not, because your cursor would only change when hovering over the resizable ones. Now, a new distraction is created when hovering over non-resizable edges by changing the cursor.

If we change the cursor, it should be indicative of something the user _can_ do, not something they _can't_ do.

Reply:
For me, when i try to resize a window, if the cursor doesn't change, i feel like
i somehow *missed* the window border.

> If we change the cursor, it should be indicative of something the user can do,
> not something they can't do.

I argue that "negative feedback" is better than "no feedback."

---

We need a third vote of course.

Reply:
I don't like this cursor change either. I can't say I've ever had this as a problem before.

Having it for `Super+Secondary-click` doesn't seem too bad - the user is clearly trying to do something so it could be good to tell them why it doesn't work. But showing it when you're just hovering over a window border feels aggressive.

Reply:
Dropped the commit.

---

how about this cursor ![image](https://github.com/SerenityOS/serenity/assets/70647861/2bd494bf-1af6-41e8-95de-796bb9f99473)? (but this would add 3 new cursors just for single use)

Reply:
Thanks! I'd rather not have any cursor change at all; regardless of the cursor shown.